docs/content/videos.rst

```diff
@@ -0,0 +1,22 @@
+Videos
+======
+
+A curated list of various types of video content:
+
+Tutorials
+---------
+
+Step-by-step and easy to follow tutorials. Coming soon!
```

Comment:
Not sure this needs a description. I think title is pretty self-explanatory. Same for the other two sub-sections.